feat(react-icons): reexport tabler icons

gdostie opened the pull request:
### Proposed Changes

`@coveord/plasma-react-icons` will now re-export every tabler icon. This way we won't need to install `@tabler/react-icons` ourselves in all our packages.

Note: I removed the list of plasma icons from the iconography page on the plasma website and added a link to tabler's website instead.

### Potential Breaking Changes

<!-- List all changes that might be breaking to plasma's users if any. -->

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

GermainBergeron commented:
>Note: I removed the list of plasma icons from the iconography page on the plasma website and added a link to tabler's website instead.

Did you talk with UX about their point of view on the old icons? Will they continue to use some of them?

gdostie commented:
> > Note: I removed the list of plasma icons from the iconography page on the plasma website and added a link to tabler's website instead.
> 
> Did you talk with UX about their point of view on the old icons? Will they continue to use some of them?

They have identified some icons that will continue to be in use since they haven't found any replacement for them in tabler, such as the crawling bot icon for crawling modules or the query pipelines one. They don't have any objection against removing it from the plasma website since they don't refer to it, so its up to us

gdostie commented:
I added back the old icons list inside a collapsed accordion, lmk if this is good with you.
